### DISCUSSION TITLE
Online . editor can't open branches with non-latin characters

### DISCUSSION THREAD

**Discussion by Katsute** [community/community#5229] [Codespaces] [answered]
Using the `.` command to open the online editor doesn't work if the branch contains encoded characters (ex: `@`)

**Comment by bamurtaugh** [comment]
cc @eamodio for thoughts

**Comment by Katsute** [comment]
If I go directly to a branch directly it works correctly
`https://github.dev/KatsuteDev/JCore/tree/enforce-timeouts@443b11e` (`enforce-timeouts@443b11e` is the branch)

the issue seems to be that when you go on GitHub the branch name is encoded (`@` → `%40`) which works properly on GitHub but the dev site seems to have issues resolving the branch
`https://github.dev/KatsuteDev/JCore/tree/enforce-timeouts%40443b11e` (there is no such branch `enforce-timeouts%40443b11e`)

**Comment by bamurtaugh** [comment]
cc @joyceerhl

**Comment by bamurtaugh** [comment]
Filed https://github.com/microsoft/vscode-remote-repositories-github/issues/78 to track the issue in the appropriate repo

**Comment by joyceerhl** [comment, marked as answer]
This should be fixed now. @Katsute can you confirm?

**Reply by Katsute** [reply]
LGTM 👍
